charts/kubermatic/static/store-container-new.yaml

```diff
@@ -0,0 +1,34 @@
+# This file has been generated using hack/update-kubermatic-chart.sh, do not edit.
+
+name: store-container
+image: d3fk/s3cmd@sha256:2061883abbf0ebcf0ea3d5d218558c9c229f212e9c08af4acdaa3758980eb67a
+command:
+- /bin/sh
+- -c
+- |
+  set -e
+  s3cmd --access_key=$ACCESS_KEY_ID --secret_key=$SECRET_ACCESS_KEY --host=$ENDPOINT --host-bucket='%(bucket).'$ENDPOINT put /backup/snapshot.db s3://$BUCKET_NAME/$CLUSTER-$BACKUP_TO_CREATE
```

Comment:
Our current defaults to the same behavior as s3cmd's `--no-check-certificate` option. So we should should default to it here, too. It would be awesome if we could make that configurable as well!

Reply:
Done. (not configurable for now)